### PR TITLE
Clustering cache replication

### PR DESCRIPTION
**Pull request by jhinkey**
Hi Ray,

Can you review these clarifications to our Clustering docs regarding cache replication?

See https://github.com/rotty3000/liferay-docs/commit/d76e69dd2ffaadba4c73f80f98cbd6265a3e6c9a

Basically, I state that replication is not done via copy.

### CODE REVIEW

**Comment by rotty3000** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: discover/deployment/articles/03-configuring-liferay/03-liferay-clustering.markdown [hunk 1]
```diff
@@ -187,70 +187,41 @@ clusters.
...
+
+By default Liferay does not copy cache changes between nodes. If a user deletes
+or changes a cached entity, for example, Cluster Link sends a *remove* message
+to the other nodes to invalidate the entity in their cache. Requesting that
+entity from one of these caches results in a *miss*; the entity is then
+retrieved from the database and put into that cache. Similarly, entities added
+to one node's cache are not copied to caches on the other nodes. An attempt to
+retrieve the new entity from a cache that doesn't have it results in a *miss*.
+The miss triggers Cluster Link to retrieve the entity from the database and
+store it in that cache.
```
Comment (rotty3000) [inline discover/deployment/articles/03-configuring-liferay/03-liferay-clustering.markdown]: Slightly different wording.
```
By default Liferay does not copy cached entities between nodes. If an entity is deleted
or changed, for example, Cluster Link sends an *remove* message
to the other nodes to invalidate this entity in their local cache. Requesting that
entity on another node results in a cache *miss*; the entity is then
retrieved from the database and put into the local cache. Entities added
to one node's local cache are not copied to local caches of the other nodes. An attempt to
retrieve a new entity on a node which doesn't have that entity cached results in a cache *miss*.
The miss triggers the node to retrieve the entity from the database and
store it in its local cache.
```

**Comment by jhinkey** [comment]
@rotty3000 I applied your rewrite of that paragraph and copied the changes to the DXP clustering doc 847bca6 too. All good?

**Comment by jhinkey** [comment]
Sent https://github.com/sez11a/liferay-docs/pull/2861

**Pull request closed without merging**